src/openms/include/OpenMS/ANALYSIS/ID/PeptideProteinResolution.h

```diff
@@ -53,10 +52,16 @@ namespace OpenMS
     std::set<Size> pep_indices;
     
     // Default constructor
-    ConnectedComponent();
+    ConnectedComponent() = default;
+
+    // Copy constructor
+    ConnectedComponent(const ConnectedComponent&) = default;
+
+    // Copy assignment
+    ConnectedComponent& operator=(const ConnectedComponent&) = default;
```

Comment:
if you add those, why not add move constructor/assignment as well?

Reply:
I just added them because there was the default one. Probably better just to remove all and we get the move for free.